PartOfModule1/PartOfModule/Program.cs

```diff
@@ -0,0 +1,161 @@
+﻿/* Our task is:
+Use StyleCope in a project.
+Create and array with N elements, where N is specified from the console line.
+Fill it with random numbers from 1 to 26 inclusive.
+Create 2 arrays, where in first array there will be only even values, and in the second - odd.
+Replace numbers in array 1 and 2 with letters of English alphabet.
+The cell values of these arrays are equal to the ordinal numbers of each letter in the alphabet.
+If the letter is one of the list (a, e, i, d, h, j) then it must be in uppercase.
+Display the result of which array contains more uppercase letters. Print both arrays to the screen.
+Each of the arrays should be displayed in 1 line, where its values will be separated.
+*/
+namespace PartOfModule
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            // size of our future array
+            int n = 0;
+            Console.WriteLine($"Write an integer number to console to define a size of an array");
+            string str = Console.ReadLine();
+
+            // we need to exclude accidents when we miswrite or type characters instead of number
+            bool isNumeric = int.TryParse(str, out n);
+
+            if (string.IsNullOrEmpty(str) || !isNumeric)
```

Comment:
In this case if **n** is not a number we need either ask user to input correct value or stop the program, as we are unable to proceed w/o the correct input